docs/ref/middleware.txt

```diff
@@ -158,12 +158,22 @@ It will NOT compress content if any of the following are true:
 You can apply GZip compression to individual views using the
 :func:`~django.views.decorators.gzip.gzip_page()` decorator.
 
+.. note::
```

Comment:
I try to avoid usage of ".. note::" in favor of ".. admonition:: some descriptive title". In this case, I don't think a call out box is necessary.